packages/es-dev-server/README.md

```diff
@@ -39,7 +39,7 @@ npx es-dev-server
 ### Server configuration
 | name                 |  type          | description                                                                |
 | -------------------- | -------------- | -------------------------------------------------------------------------- |
-| port                 | number         | The port to use. Default: 8080                                             |
+| port                 | number         | The port to use, uses a random free port if not set.                       |
```

Comment:
so if I restart the server I need to update my url in the browser? 😭 
also I can no longer bookmark the default url?

or does it use 8080 by default and if not free picks another random free one?

Reply:
It uses 8000 by default, if that one is taken it picks 8001 etc. So if you restart it will stay on the same port.

Reply:
It uses 8000 by default, if that one is taken it picks 8001 etc. So if you restart it will stay on the same port.